src/utils/config_utils/src/config_utils.c

```diff
@@ -57,13 +58,87 @@ static const char* CONFIG_RUN_AS = "runas";
 static const char* CONFIG_CONNECTION_SOURCE = "connectionSource";
 static const char* CONFIG_CONNECTION_TYPE = "connectionType";
 static const char* CONFIG_CONNECTION_DATA = "connectionData";
+static const char* CONFIG_CONNECTION_X509_CERT_FILE_PATH = "connectionX509CertFilePath";
+static const char* CONFIG_CONNECTION_X509_PRIVATE_KEY_FILE_PATH = "connectionX509PrivateKeyFilePath";
+static const char* CONFIG_CONNECTION_X509_CA_CERT_FILE_PATH = "connectionX509CaCertFilePath";
 static const char* CONFIG_ADDITIONAL_DEVICE_PROPERTIES = "additionalDeviceProperties";
 static const char* CONFIG_AGENTS = "agents";
 
 static const char* INVALID_OR_MISSING_FIELD_ERROR_FMT = "Invalid json - '%s' missing or incorrect";
 
 static void ADUC_AgentInfo_Free(ADUC_AgentInfo* agent);
 
+
+static char* ADUC_AgentInfo_Read_X509_File(const char* const x509_file_path) {
+    char* buffer = NULL;
+    long ftellResult = 0;
+    size_t bufferLength = 0;
+    size_t bytesRead = 0;
+    FILE* x509File = NULL;
+    
+    x509File = fopen(x509_file_path, "rb");
+    if (x509File)
+    {
+        if (fseek(x509File, 0, SEEK_END) == 0) {
```

Comment:
Style consistency issue: Opening brace should be on next line

Touch every file in the current set of commits, `git add` them to git staging, and then run `scripts/clang-format.sh`. see: https://github.com/Azure/iot-hub-device-update/blob/0f041bfc6cd25593ed61d58e4113b2a4dad7394f/scripts/clang-format.sh#L10

Longer-term, we need to add clang-format.sh (and some other things) to the git pre-commit hook.  Currently, the existing git pre-commit hook only runs sh-format.sh (https://github.com/Azure/iot-hub-device-update/blob/0f041bfc6cd25593ed61d58e4113b2a4dad7394f/scripts/githooks/pre-commit.sh#L29)